src/modules/UI/scenes/SendConfirmation/SendConfirmationOptions.js

```diff
@@ -31,7 +33,7 @@ export default class SendConfirmationOptions extends Component {
             </Text>
```

Comment:
The 'fee' 'help' and 'max' strings should be referencing a constant instead of being stand alone strings. 
Example Constants.FEE.   If they don't already exist , they need to be created.